regulation-worker/internal/model/model.go

```diff
@@ -36,8 +36,7 @@ type Job struct {
 
 type UserAttribute struct {
 	UserID string
-	Phone  *string
-	Email  *string
+	Opts   map[string]string
```

Comment:
In terms of readability, the following approaches sound better to me:

(Opts seems like user options, which is not exactly what we expect here to be)

```go
type User struct {
	ID string
	Attributes   map[string]string
}
```

Or 
```go
type UserAttribute struct {
	UserID string
	Optional   map[string]string
```

```go
type UserAttribute struct {
	UserID string
	KeyValue   map[string]string
```